enterprise/backend/manage.sh

```
@@ -3,4 +3,6 @@
 DJANGO_DIR=../../backend
 ENTERPRISE_SETTINGS=enterprise_core.settings
 
+export DJANGO_BASE_DIR="$(cd "$(dirname "$0")" && pwd)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)